kod/object/item/passitem/ring/ringinv.kod

```diff
@@ -62,7 +63,12 @@ messages:
 	      return False;
       }
 
-      propagate;
+      if send(poOwner,@GetKarma) > -50  
+      { 
+         send(poOwner,@MsgSendUser,#message_rsc=ringinvisibility_cant);
```

Comment:
"Send" should be written with upper-case S. If you have the Notepad++ plugin, you'll notice if you write any C calls without the upper-case letter at the start, they don't get colored.

Also, this message (ReqUse()) still needs to propagate after this new if statement.